src/renderer/components/dock/log.store.ts

```diff
@@ -146,6 +146,16 @@ export class LogStore {
     return stamp.toISOString();
   }
 
+  splitOutTimestamp(logs: string): [string, string] {
+    const extraction = /^(\d+\S+) (.*)/gm.exec(logs);
+
+    if (extraction.length < 3) {
+      return ["", ""];
+    }
```

Comment:
exec can return null
```suggestion
    if (!extraction || extraction.length < 3) {
      return ["", ""];
    }
```